code/datums/id_trim/jobs.dm

```diff
@@ -626,6 +626,35 @@
 		)
 	job = /datum/job/lawyer
 
+/datum/id_trim/job/maintenance_technician
+	assignment = "Maintenance Technician"
+	trim_state = "trim_mainttechnician"
+	department_color = COLOR_ENGINEERING_ORANGE
+	subdepartment_color = COLOR_ASSISTANT_GRAY
+	sechud_icon_state = SECHUD_MAINTENANCE_TECHNICIAN
+	minimal_access = list(
+		ACCESS_AUX_BASE,
+		ACCESS_CONSTRUCTION,
+		ACCESS_ENGINEERING,
+		ACCESS_ENGINE_EQUIP,
+		ACCESS_EXTERNAL_AIRLOCKS,
+		ACCESS_MAINT_TUNNELS,
+		ACCESS_MECH_ENGINE,
+		ACCESS_MINERAL_STOREROOM,
+		ACCESS_TECH_STORAGE,
+		)
```

Comment:
unindent all these in the file

Reply:
Alright, but you owe my GBP for code improvement then haha

Reply:
youll already get it from sprites but sure